lib/assert.ts

```diff
@@ -28,9 +28,17 @@ import path from 'path';
 import {parse} from './stacktrace.js';
 import {isString} from './common-utils.js';
 
-function check_path(parent: string, directory: string) {
+function removeFileProtocol(path: string) {
+    if (path.startsWith('file://')) {
+        return path.slice('file://'.length);
```

Comment:
can you make this a const fileprefix='file://'

Reply:
Good idea, thank you